### DISCUSSION TITLE
What are the `qBitrr-` tags

### DISCUSSION THREAD

**Discussion by Minituff** [Feramance/qBitrr#99] [General]
Hello,

I just got Qbittr up and running and I am starting to see some new tags pop up on my torrents

-  `qBitrr-allowed_seeding`
-  `qBitrr-allowed_stalled`
-  `qBitrr-ignored`
-  `qBitrr-imported`

I don't see these tags anywhere in the config. I'm not too sure what they do, or how I can configure them and/or disable them.

**Comment by Feramance** [comment]
Hi, 

Simply put the below are used by qBitrr to handle automation. They are not to be used manually:

- `qBitrr-allowed_seeding`
- `qBitrr-allowed_stalled`
- `qBitrr-imported`

Then `qBitrr-ignored` is used when you want to disallow qBitrr from handling the torrent (pausing, resuming, deleting, etc)
